docs/02-app/02-api-reference/06-create-next-app.mdx

```diff
@@ -26,7 +26,10 @@ pnpm create next-app
 ```
 
 ```bash filename="Terminal"
+# Both methods works Identically
 bunx create next-app
+or
+bunx create-next-app
```

Comment:
```suggestion
bunx create-next-app
```

Reply:
Let's leave it as this as from testing against latest bun this appears to work correctly while the other doesn't